docs/README.md

```diff
@@ -0,0 +1,68 @@
+### Configuration
+Supported are command line argument and environment variables for configuration. Command line arguments will take precedence over the environment variables. 
+* Command line
+
+        python -m rpi_sensors_exporter [-h] [-p PORT] [-c CONFIG_FILE] [-v] [-d]
+
+
+| Option |Default| Description |Required|Since version|
+|--|--|--|--|--|
+| -p, --port |8080| The port number on which the exporter will run |No| 0.0.1 |
+| -c, --config_file |N/A| Path to the exporter's configuration file (*please refer to "Configuration file format" section for more information*) |No |0.0.1|
+| -v, --verbose |N/A| Enables exporter's INFO logs on stdout |No |0.0.1|
+| -d, --debug | N/A | Enables exporter's INFO and DEBUG logs on stdout | No | 0.0.1|
+| -h, --help | N/A | Prints all supported arguments|No|0.0.1
+
+* Environment variables
+
+The following environment variables are supported
+
+| Option |Default| Description |Required|Since version|
+|--|--|--|--|--|
+| SENSORS_EXPORTER_PORT |8080| The port number on which the exporter will run |No| 0.0.1 |
+| SENSORS_EXPORTER_CONFIG |N/A| Path to the exporter's configuration file (*please refer to "Configuration file format" section bellow for more information*) |No |0.0.1|
+
+
+### Configuration file format
+YAML formatted document with optional configuration for sensors connected to GPIO or ADC:
+
+| Configuration | Description |Type|Required|Since version|
+|--|--|--|--|--|
+| gpio_devices | A list of dictionaries describing GPIO devices connected to the system |list|No| 0.0.1 |
+| gpio_devices.name | Name of the sensor |str|Yes| 0.0.1 |
+| gpio_devices.type | Type of the sensor |str|Yes| 0.0.1 |
+| gpio_devices.pin | GPIO pin on which the sensor is connected to the system, should be between 0 and 27 inclusive |int|Yes| 0.0.1 |
+| ads_devices | A list of dictionaries describing devices connected to the ADC (Analogue-Digital Converter) |list|No| 0.0.1 |
+| ads_devices.name | Name of the sensor |str|Yes| 0.0.1 |
```

Comment:
> Fix ADS var names

No changes are needed since for now only ADS1115 ADC is supported. If support for other type of ADC is added in the future it will become impossible to make a difference between them. Code is changed, so this will be more obvious.

Reply:
> Also, mention the importance and meaning of name/type for adc sensors

Doc is extended, configuration validation is introduced.